pkg/decode/decode.go

```diff
@@ -11,10 +11,18 @@ import (
 
 type unmarshalFunc func([]byte, interface{}) error
 
+// FromJSON reads an asyncapi document from an input in a JSON format
+// and stores it in the value. If operation fails function returns an error.
+//
+// See InvalidProperty, InvalidDocument, UnsupportedAsyncapiVersion in pkg error.
 func FromJSON(v interface{}, reader io.Reader) error {
 	return json.NewDecoder(reader).Decode(&v)
 }
 
+// FromYaml reads an asyncapi document from an input in yaml format
+// and stores it in the value. If operation fails it returns error.
```

Comment:
```suggestion
// and stores it in the value. If the operation fails, the function returns an error.
```
And here: I'm wondering if it is a value or the value - do you mean any value or a particular one?